### PR TITLE
Cleaned up code

### PR DESCRIPTION
**Pull request by maaverik**

### CODE REVIEW

File: simplq/src/components/pages/Join/index.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import JoinQueueForm from './Form';
 import { setTokenId, setJoinerStep, setAheadCount, setQueueId } from '../../../store/appSlice';
 import * as TokenService from '../../../services/token';
 import { handleApiErrors } from '../../ErrorHandler';
-import { SimplQHeader } from '../../common/Header.stories';
+import { SimplQHeader } from '../../common/Header/Header.stories';
```
Comment (daltonfury42) [inline simplq/src/components/pages/Join/index.jsx:7]: Here it should be just `import { SimplQHeader } from '../../common/Header'`

File: simplq/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint src --ext .js,.jsx --fix-dry-run",
+    "lint": "eslint src --ext .js,.jsx --fix",
```
Comment (daltonfury42) [inline simplq/package.json:34]: Do we need these filters by extension? Otherwise, should we add json too?

same for the lint-staged one.

Comment (daltonfury42) [reply simplq/package.json:34]: Remove fix-dry-run from lintstaged too, line 79

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -3,6 +3,11 @@ name: Lint
 on: 
   pull_request:
       branches: [master]
```
Comment (daltonfury42) [inline .github/workflows/lint.yml:5]: add new-frontend and prod also to this

File: simplq/package.json (hunk 2 above)
Comment (maaverik) [reply simplq/package.json:34]: It didn't run on all files when I tried using the glob syntax (src/**/**), that's why I used this.

**Comment by maaverik** [comment]
Looks like the extra brace is caused by the removing the fix-dry-run somehow. Maybe adding json will help.

**Comment by maaverik** [comment]
It might be a conflict between eslint and prettier during the fix. Could you look into this?

**Comment by daltonfury42** [comment]
ok. lets merge this one.

**Pull request merged**